tests/integration/synapseutils/test_synapseutils_sync.py

```diff
@@ -234,46 +235,10 @@ def test_write_manifest_data__unicode_characters_in_rows():
         assert_equals(datarow['col_B'], dfrow.col_B)
 
 
-def test_syncFromSynapse():
```

Comment:
why was this deleted?

Reply:
It's shadowing another test of the same name:
https://github.com/Sage-Bionetworks/synapsePythonClient/blob/67db15cf5f32fcb4e7953396df192712f82ff733/tests/integration/synapseutils/test_synapseutils_sync.py#L113